store/config_test.go

```diff
@@ -1,19 +1,79 @@
-package store_test
+package store
 
 import (
 	"math/big"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
-	strpkg "github.com/smartcontractkit/chainlink/store"
 	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zapcore"
 )
 
 func TestStore_ConfigDefaults(t *testing.T) {
 	t.Parallel()
-	config := strpkg.NewConfig()
+	config := NewConfig()
 	assert.Equal(t, uint64(0), config.ChainID)
 	assert.Equal(t, *big.NewInt(20000000000), config.EthGasPriceDefault)
 	assert.Equal(t, "0x514910771AF9Ca656af840dff83E8264EcF986CA", common.HexToAddress(config.LinkContractAddress).String())
 	assert.Equal(t, *big.NewInt(1000000000000000000), config.MinimumContractPayment)
 }
+
+func TestStore_addressParser(t *testing.T) {
```

Comment:
We've so far avoided testing unexported methods, essentially treating them like private methods in regular OO. These seem like general methods that could be reused in other places. Maybe move them to the utils package and make them exported?

Reply:
This function is only useful for `parseEnv`, due to the fairly restricted type signature. Testing "private" methods is fairly common in go projects though, so I feel like we should consider adopting this convention.

Reply:
Summarizing a larger discussion: It's worth testing unexported variables because you want them tested and don't want to increase the surface area of your public API just to test something.